src/memory.wasm-gc.mbt

```diff
@@ -1,18 +1,18 @@
-///|
+///| Memory is a pointer to a block of memory.
 type Memory[_] Int
 
-///|
+///| convert pointer to int
```

Comment:
Can you please format your documentations for function so that they:

1. Start with a verb that describes what the function does in its third-party singular form. For simple getter/attribute use `Returns xxx`.
2. Start with a capitalized character.